### PR TITLE
Implement gui functionality

### PR DESCRIPTION
**Pull request by AndersBensen**
Rasmus and Anders made and added pictures for every room, made the steps show in the gui and made a background for the whole gridpane.

### CODE REVIEW

File: src/gui/GameController.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import acq.IGame;
 import java.net.URL;
 import java.util.ResourceBundle;
+import java.util.concurrent.TimeUnit;
```
Comment (AndersBensen) [inline src/gui/GameController.java]: delete

File: src/gui/GameController.java [hunk 2]
```diff
@@ -16,24 +17,23 @@
 import javafx.scene.image.ImageView;
 import javafx.scene.input.KeyEvent;
 import javafx.scene.layout.GridPane;
+import static javafx.util.Duration.millis;
```
Comment (AndersBensen) [inline src/gui/GameController.java]: delete

File: src/gui/GameController.java (hunk 1 above)
Comment (AndersBensen) [reply src/gui/GameController.java]: fixed in: 31e8dcbe2c3bf879c96fa749765326d69c393f1b

File: src/gui/GameController.java (hunk 2 above)
Comment (AndersBensen) [reply src/gui/GameController.java]: fixed in: 31e8dcbe2c3bf879c96fa749765326d69c393f1b

**Review by muggell — APPROVED** [review]

**Pull request merged**